src/NetPanzer/Views/MainMenu/MenuTemplateView.cpp

```diff
@@ -241,7 +242,10 @@ void MenuTemplateView::doDrawM(Surface &viewArea,
     }
     //        clientArea.BltRoundRect(MenuRect, 10,
     //        Palette::gray256.getColorArray());
-    clientArea.FillRoundRect(MenuRectStart, 3, Color::white);
+    
+    // TODO: check config loaded OK
+    bool okidoki = ScriptManager::loadSimpleConfig("scripts/mainmenu.lua");
```

Comment:
Looks like most of the error checking is done in ScriptManager::loadSimpleConfig(), and the LOGGER outputs a message. We could just exit here with no extra message if false is returned.